dockerfiles/Dockerfile.env.dist

```diff
@@ -0,0 +1,3 @@
+POSTGRES_NAME=StreamWebs
```

Comment:
No reason. I was just keeping to the capitalization of StreamWebs. But I can change it since I realized I forgot to update this file...